css/filmstrip/_vertical_filmstrip.scss

```diff
@@ -136,6 +136,13 @@
         display: flex;
         transition: opacity 1s;
     }
+
+    .hide-scrollbar#filmstripRemoteVideos {
+        margin-right: 7px; // Scrollbar size
+        &::-webkit-scrollbar {
```

Comment:
Out of curiosity - why hide the scrollbar exactly?

Reply:
The user has no way to move it (they are on a SIP client) so it looks odd.